README.md

```diff
@@ -1 +1,260 @@
+
+> Open the index.html file to view the contents 
+
+
+### Prerequisites
+
+In order to run this project you need:
+- Install Git in your computer [Git](https://git-scm.com/downloads)
+- Any text editor [VSCode](https://code.visualstudio.com/download) 
+- Install linters on your local env't
+ 
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder, access the directory and run the html file: 
+To clone: git clone git@github.com:Yosefgeda/My-portfolio.git
+to change to directory: cd My-portfolio
+Inside you will find the html file.
+
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Linters have been installed for this project(npm install).
+Added a gitignore file and included the node_modules and text.md files.
+Linter actions are integreated witg github.
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+execute the following steps:
+To run the project on the html file run it through liveserver.
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+ run npx hint . 
+ and npx stylelint "**/*.{css,scss}"
+
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@yosefgeda](https://github.com/yosefgeda)
+- Twitter: [@yosefgeda](https://twitter.com/yosegeda)
+- LinkedIn: [yosefgeda](https://linkedin.com/in/yosefgeda)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[More_section_will_be_included]**
+- [ ] **[The_coming_features_will_include_previous_works_done]**
+
+<!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issue page](https://github.com/Yosefgeda/My-portfolio/issues)
+
+<!-- SUPPORT -->
+
+If you like this project follow and give thumbs up on likedin
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to my coding partener Amanuel Belete.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[What_version_of_chrome_do_I_need_to_view_the_page]**
+
+  - [All_versions_are_capable]
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is currently unlicensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [Optional] The LICENSE file was included in the [TEMPLATE](https://github.com/microverseinc/readme-template/blob/master/MIT.md), but you omitted it. Please add it. The license is open-source, and the aim is to help you document your projects in the best way ever and professionally with industry standards.